<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmsManagementWidget</class>
 <widget class="QWidget" name="AlarmsManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarms</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxQuery">
     <property name="title">
      <string>Query</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="labelPeriod">
          <property name="text">
           <string>Period:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxPeriod"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="labelStartTime">
          <property name="text">
           <string>Start Time:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QDateTimeEdit" name="dateTimeEditStartTime">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1970</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string notr="true">yyyy/MM/dd H:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="labelEndTime">
          <property name="text">
           <string>End Time:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QDateTimeEdit" name="dateTimeEditEndTime">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelAssetNO">
          <property name="text">
           <string>Asset NO.:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditAssetNO">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelType">
          <property name="text">
           <string>Type:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="comboBoxType"/>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="labelAcknowledged">
          <property name="text">
           <string>Acknowledged:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QComboBox" name="comboBoxAcknowledged"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="toolButtonQuery">
          <property name="toolTip">
           <string>Query</string>
          </property>
          <property name="statusTip">
           <string>Query</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../controlcenter.qrc">
            <normaloff>:/icon/resources/images/querydatabase.png</normaloff>:/icon/resources/images/querydatabase.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonFilter">
          <property name="toolTip">
           <string>Filter</string>
          </property>
          <property name="statusTip">
           <string>Filter</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../controlcenter.qrc">
            <normaloff>:/icon/resources/images/filter.png</normaloff>:/icon/resources/images/filter.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAlarms">
     <property name="title">
      <string>Alarms</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>120</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionAcknowledge">
   <property name="text">
    <string>Acknowledge</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../controlcenter.qrc"/>
 </resources>
 <connections/>
</ui>
